aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -194,6 +195,24 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
     }
   }
 
+  /**
+   * Validate the Glue table is Iceberg table by checking its parameters. If the table properties
+   * check does not pass, for Iceberg it is equivalent to not having a table in the catalog. We
+   * throw a {@link NoSuchIcebergTableException} in that case.
+   *
+   * @param table glue table
+   * @param fullName full table name for logging
+   * @throws NoSuchIcebergTableException if the table is not an Iceberg table
+   */
+  static void checkIfTableIsIceberg(Table table, String fullName) {
```

Comment:
Nonblocking nit: can we rename this to `validateIsIcebergTable`

Reply:
Discussed offline - `validate` may imply `ValidationException`. We'll stick with `checkIfTableIsIceberg`